src/main/java/tempest/ui/GUIManager.java

```diff
@@ -68,8 +60,8 @@ public GUIManager(State state, Supervisor supervisor) {
                 chartView = new ChartViewPage(this),
                 barChart = new BarChart(state, this),
                 lineChart = new LineChart(state, this),
-                pieChart = new PieChart(state, this)
-
+                pieChart = new PieChart(state, this),
+                dataProtection = new DataProtectionPage(state, this),
```

Comment:
Also state only needs to be passed in if its going to be used - which the data protection page - doesn't so could you not pass in state